packages/react-sdk/src/useSendMessage.ts

```diff
@@ -18,6 +24,11 @@ export const useSendMessage = (
     const { action: sendMessage, ...rest } = useAction(room, 'sendMessage', config)
 
     return {
+        /** Sends a message to the stream.
+         * @param message - The message to send.
+         * @param options - Additional options for the message.
+         * @returns The event id of the message.
+         */
```

Comment:
I couldn't find a way to reference and show the tsdoc that we have on `Channel.sendMessage` 🥲